packages/orchestration/src/cosmos-api.ts

```diff
@@ -29,25 +32,34 @@ export type CosmosValidatorAddress = ChainAddress & {
   addressEncoding: 'bech32';
 };
 
+/** Info for an IBC Connection (Chain:Chain relationship, that can contain multiple Channels) */
```

Comment:
```suggestion
/** Info for an IBC Connection (Chain:Chain relationship that can contain multiple Channels) */
```

> A restrictive clause is not offset with commas. -- [Restrictive Clause: Explanation and Examples](https://www.grammar-monster.com/glossary/restrictive_clause.htm)

or change "that" to "which" for a non-restrictive clause:

```suggestion
/** Info for an IBC Connection (Chain:Chain relationship, which can contain multiple Channels) */
```

I'm cursed with a brain that throws syntax errors on English grammar just as much as programming language grammar. :-/

Reply:
Thank you - I appreciate your appreciation of grammar!